openshift/ci-operator/Dockerfile.in

```diff
@@ -1,5 +1,5 @@
 # Do not edit! This file was generated via Makefile
-FROM registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.21-openshift-4.16 as builder
+FROM registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17 as builder
```

Comment:
Is there no RHEL 8 variant with 1.22? Because productized images will be still RHEL 8 based. Otherwise fine

Reply:
There is `registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17` I can change to that.

Reply:
done